src/RapidezServiceProvider.php

```diff
@@ -12,6 +12,8 @@
 use Illuminate\Support\Facades\View;
 use Illuminate\Support\Facades\Vite;
 use Illuminate\Support\ServiceProvider;
+// TODO: Anyone a better name for this?
+use Illuminate\View\View as ViewView;
```

Comment:
FacadeView for the first one and then just View for this one? 😇

Reply:
Yeah but are we going to do that for all facades? Facades are used more compared to macroable classes

Reply:
Could always rename it to `MacroView` if it's only used as a macroable